pkg/option/config.go

```diff
@@ -89,6 +89,10 @@ type daemonConfig struct {
 
 	// AgentLabels contains additional labels to identify this agent in monitor events.
 	AgentLabels []string
+
+	// DisableHostIPRestore disables restoring the host IPs based on state
+	// left behind by previous Cilium runs.
+	DisableHostIPRestore bool
```

Comment:
Could you put this near the Restore option further up? they seem more related to each other conceptually. Alternatively, near the HostV*Addr fields, since it impacts them.